### PR TITLE
Add some debug info to the Dangerfile 

### PR DESCRIPTION
**Pull request by orta**
when you include the word verbose in the message body.

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // @flow
 
-// import { danger, warn } from "danger"
+import { danger, warn, markdown } from "danger"
```
Comment (macklinu) [inline dangerfile.js]: This was commented out, but now it's not. I remember in #54 (https://github.com/danger/danger-js/commit/0ba01b15e56093c5ce11ecccccef738b3ef66cf0#diff-3abee5fb96982a8bed436d8b59267dd8R76) that the danger DSL was added to the global namespace for convenience, at least as I understood it. Is this still the case?

Comment (orta) [inline dangerfile.js]: It is the case, yes, but its fakey - this is in there for tooling -> https://github.com/danger/danger-js/blob/master/source/runner/DangerfileRunner.js#L90

This means that flow/TS will give you the right autocompletions + documentation. The runner will just comment it out before evaling the DAngerfile

**Comment by orta** [comment]
Had a debate about having this be a part of danger, which can be triggered by a body message instead  of just adding it to this specific Dangerfile - think I'm gonna do that

File: dangerfile.js (hunk 1 above)
Comment (macklinu) [inline dangerfile.js]: Ah makes sense regarding the Flow/TS types. 👍  thanks

**Comment by macklinu** [comment]
@orta what are next steps for this PR? Are you suggesting adding this as a feature of Danger-js itself?

**Comment by orta** [comment]
Yeah, I was debating this as something which you could say inside the GitHub PR body like `[danger debug]` which would output the above, wanted to keep it open till I or someone else made up their mind about it 💃

**Comment by orta** [comment]
OK, I'm closing this. Happy for someone to pick up but TBH I think `danger pr` is a better call

**Pull request closed without merging**